packages/react-storage/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './StorageManager';
```

Comment:
I'd like to move away from `export *` syntax for top level exports, way too easy for types to get exported since they aren't captured in our export snapshots

Reply:
word